Add 'Scope based Authorization for Internal REST APIs doc' (#1447)
wso2/docs-is: en/docs/develop/scope-based-authorization-for-internal-rest-apis.md

# Scope based Authorization for Internal REST APIs

WSO2 Identity Server supports API authentication using 
OAuth2 common flows, where users can obtain a token using an 
oauth2 flow and use it to invoke the API.

Authorization for the APIs in WSO2 Identity Server is enforced
at the endpoint level using **permissions**. Each secured endpoint
has a predefined minimum level of permission that is required to be
able to consume the endpoint. In order to access a particular endpoint,

> the user has to belong to a **role** that is in or above the defined

Reviewer: ```suggestion
the user has to belong to a **role** that includes the defined
```

> permission level. WSO2 Identity Server now supports scope based API

Reviewer: ```suggestion
the user has to belong to a **role** that includes the defined
permissions. WSO2 Identity Server now supports scope based API
```
